tools/rush-tools-new.ts

```diff
@@ -0,0 +1,276 @@
+import * as path from "path";
```

Comment:
Removed old `rush-tools` file and renamed `rush-tools-new` to `rush-tools`.
Fixed: https://github.com/SimplrJS/react-forms/commit/479b2962a32ff9177c364dac425ed8a21ed4a6ee